include/worker.hpp

```diff
@@ -26,8 +30,14 @@ class Worker
 
     /**
      * @brief Constructor.
+     *
+     * Constructor will also, based on symlink pick the correct JSON and
+     * initialize the parsed JSON variable.
+     *
+     * Note: Throws std::exception in case of construction failure. Caller needs
+     * to handle to detect successfull object creation.
```

Comment:
nit

Reply:
What is the comment here?

Reply:
Not significant, just the spelling of successful. just for info.

Reply:
Done.